lib/config/options/index.ts

```diff
@@ -298,10 +298,10 @@ const options: RenovateOptions[] = [
   {
     name: 'composerIgnorePlatformReqs',
     description:
-      'Enable / disable use of --ignore-platform-reqs in the Composer package manager.',
-    type: 'boolean',
-    default: true,
-    globalOnly: true,
```

Comment:
@viceice I just saw that with my latest master merge this option (didn't know what its purpose was before, as it was `admin`) this option should be disabled in my opinion as you would want to be able to configure the excluded composer requirements on repository level. Is there anything else I need to adjust?

Reply:
This is / was a global (previously admin) option, so can only be set by bot admins.

@rarkins Can we open this option for repo config? I don't see a huge risk here, needs proper quotes as i suggested below

Reply:
OK